NOTICE.txt

```diff
@@ -1494,6 +1494,50 @@ Files covered:
 spyder/images/winpython.svg
 
 
+-------------------------------------------------------------------------------
+
+
+
+OOjs UI MediaWiki lib
+---------------------
+
+Copyright (c) 2011-2018 OOUI Team and other contributors.
+
+
+Principal Authors:
+Bartosz Dziewoński <matma.rex@gmail.com>
+Ed Sanders <esanders@wikimedia.org>
+James D. Forrester <jforrester@wikimedia.org>
+Kirsten Menger-Anderson <kmenger@wikimedia.org>
+Kunal Mehta <legoktm@gmail.com>
+Moriel Schottlender <moriel@gmail.com>
+Prateek Saxena <prtksxna@gmail.com>
+Roan Kattouw <roan@wikimedia.org>
+Rob Moen <rmoen@wikimedia.org>
+Timo Tijhof <krinklemail@gmail.com>
+Trevor Parscal <trevor@wikimedia.org>
+Volker E. <volker.e@wikimedia.org>
+
+Source/Download: https://phabricator.wikimedia.org/diffusion/GVED/browse/master/lib/oojs-ui/themes/mediawiki/images/icons/
+License: MIT (Expat) License | https://opensource.org/licenses/MIT
+
+Modifications made: Removed parentheses, make it bigger.
```

Comment:
The actual line is fine, perfect in fact. 

With regard to the parenthesis removal though, I'd personally prefer it with the parens. It helps it avoid being misread as "glob" (which the simple mode at least kinda sorta looks like, with a ``*.`` as part of the default strings) rather than "regex" (which I'd be uncertain about it if I didn't actually know the truth), as well as seems more standard, judging from the icons on Wikimedia commons (where you got this) and the first hits on Google images, and looks a little cleaner and more self-contained. But its not the end of the world.

Reply:
I removed the parenthesis because `.*` is also used to denote regexps in Sublime and VSCode, so it's kind of universal right now.

Reply:
Okay; if its the commonly understood convention then its fine I guess.